### PR TITLE
"Added Test cases for logging identity"

### PR DESCRIPTION
**Pull request by vstratan**
@irina-mitrea-luxoft Please review the new PR for relese/5.1.0

### CODE REVIEW

File: transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/logging/WsebAcceptorUserLoggingIT.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ public void evaluate() throws Throwable {
                         .httpChallengeScheme("Basic")
                         .userPrincipalClass("org.kaazing.gateway.security.auth.config.parse.DefaultUserConfig")
                         .loginModule()
-                            .type("class:org.kaazing.gateway.transport.wseb.logging.BasicLoginModuleWithDefaultUserConfig")
+                            .type("class:org.kaazing.gateway.transport.wseb.logging.loggingmodule.BasicLoginModuleWithDefaultUserConfig")
```
Comment (irina-mitrea-luxoft) [inline transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/logging/WsebAcceptorUserLoggingIT.java]: Can we rename this to "loginmodule"?

File: transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/logging/WsebAcceptorUserLoggingMultiplePrincipalsInConfigIT.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                    base.evaluate();
+                    MemoryAppender.assertMessagesLogged(expectedPatterns,
+                            forbiddenPatterns, ".*\\[.*#.*].*", true);
+                }
+            };
+        }
+    };
+    public GatewayRule gateway = new GatewayRule() {
+        {
+            GatewayConfiguration configuration = new GatewayConfigurationBuilder()
```
Comment (irina-mitrea-luxoft) [inline transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/logging/WsebAcceptorUserLoggingMultiplePrincipalsInConfigIT.java]: Minor indentation issue

File: transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/logging/loggingmodule/BasicLoginModuleWithDefaultUserConfig.java [hunk 3]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.kaazing.gateway.transport.wseb.logging;
+package org.kaazing.gateway.transport.wseb.logging.loggingmodule;
```
Comment (irina-mitrea-luxoft) [inline transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/logging/loggingmodule/BasicLoginModuleWithDefaultUserConfig.java]: Can we rename this to "loginmodule"?

File: transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/auth/BasicLoginModuleWithMultiplePrincipalsInConfig.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    private UnixPrincipal unixPrincipal = new UnixPrincipal("unixPrincipalName");
+
+    @Override
+    public boolean commit() throws LoginException {
+        if (super.commit()) {
+            subject.getPrincipals().add(unixPrincipal);
+            return true;
+        }
+        else
+            return false;
```
Comment (irina-mitrea-luxoft) [inline transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/auth/BasicLoginModuleWithMultiplePrincipalsInConfig.java]: We could have this line between { ... } or remove the else

File: transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/logging/WsnAcceptorUserLoggingMultiplePrincipalsInConfigIT.java [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+                    base.evaluate();
+                    MemoryAppender.assertMessagesLogged(expectedPatterns,
+                            forbiddenPatterns, ".*\\[.*#.*].*", true);
+                }
+            };
+        }
+    };
+    public GatewayRule gateway = new GatewayRule() {
+        {
+            GatewayConfiguration configuration = new GatewayConfigurationBuilder()
```
Comment (irina-mitrea-luxoft) [inline transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/logging/WsnAcceptorUserLoggingMultiplePrincipalsInConfigIT.java]: Minor indentation issue

**Comment by irina-mitrea-luxoft** [comment]
@vstratan This looks ok to me. Do the tests pass successfully? (The Travis build seems to fail before the newly added tests have been run)

**Comment by irina-mitrea-luxoft** [comment]
@vstratan This looks ok to me. Do the added tests pass successfully on your machine? (The Travis build seems to fail before the newly added tests have been run)

**Comment by vstratan** [comment]
@irina-mitrea-luxoft I can confirm both on wsn and wseb - tests are passing on my local machine

File: transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/logging/WsebAcceptorUserLoggingMultiplePrincipalsInConfigIT.java (hunk 2 above)
Comment (irina-mitrea-luxoft) [inline transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/logging/WsebAcceptorUserLoggingMultiplePrincipalsInConfigIT.java]: Can we have this between formattr:iff and formatter:on?

File: transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/logging/WsnAcceptorUserLoggingMultiplePrincipalsInConfigIT.java [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+                    MemoryAppender.assertMessagesLogged(expectedPatterns,
+                            forbiddenPatterns, ".*\\[.*#.*].*", true);
+                }
+            };
+        }
+    };
+    public GatewayRule gateway = new GatewayRule() {
+        {
+            // @formatter:off
+            GatewayConfiguration configuration = new GatewayConfigurationBuilder()
```
Comment (irina-mitrea-luxoft) [inline transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/logging/WsnAcceptorUserLoggingMultiplePrincipalsInConfigIT.java]: Can we end this with formatter:on?

**Comment by irina-mitrea-luxoft** [comment]
@vstratan Hello, Vlad, just some minor feedback. If we address this minor feedback, this can be merged.

**Comment by irina-mitrea-luxoft** [comment]
Merging PR as Travis failure doesn't seem to have any connection to current PR.

**Pull request merged**